docs/execution-providers/TensorRT-ExecutionProvider.md

```diff
@@ -27,6 +27,7 @@ Pre-built packages and Docker images are available for Jetpack in the [Jetson Zo
 |ONNX Runtime|TensorRT|CUDA|
 |---|---|---|
 |master|8.4|11.4|
+|1.13|8.4|11.4|
```

Comment:
Since it's the same versions as 1.12, can we represent it in a single row, similar to how 1.7-1.8 are?